scripts/release/build/rpm/build.sh

```diff
@@ -11,13 +11,21 @@ export HOME=/root
 
 . "${HOME}"/subhome/build_env
 
-GIT_REPO_PATH=https://github.com/algorand/go-algorand
 mkdir -p "${HOME}/go/src/github.com/algorand"
-cd "${HOME}/go/src/github.com/algorand" && git clone --single-branch --branch "${BRANCH}" "${GIT_REPO_PATH}" go-algorand
-
-# Get golang 1.12 and build its own copy of go-algorand.
+cd "${HOME}/go/src/github.com/algorand"
+if ! git clone --single-branch --branch "${BRANCH}" https://github.com/algorand/go-algorand go-algorand
+then
+    echo There has been a problem cloning the "$BRANCH" branch.
+    exit 1
+fi
+
+# Get golang 1.12.9 and build its own copy of go-algorand.
 cd "${HOME}"
-python3 "${HOME}/go/src/github.com/algorand/go-algorand/scripts/get_latest_go.py" --version-prefix=1.12
+if ! curl -O https://dl.google.com/go/go1.12.9.linux-amd64.tar.gz
```

Comment:
How about setting 1.12.9 from a var? Brian already has somewhere a pending PR for go 1.13

Reply:
Let's hold off with that until after Brice finish merging his changes into the build process.
Once he'll be done with that, we should have a single location to update.